python/ray/air/_internal/usage.py

```diff
@@ -0,0 +1,18 @@
+from typing import TYPE_CHECKING
```

Comment:
I suggest you just name this ``telemetry.py``

Reply:
Let's try to call it usage or usage stats. We want to avoid the word telemetry